__tests__/app.test.js

```diff
@@ -35,12 +35,43 @@ describe('1. GET api/:path', () => {
         })
     })
     })
+
+    test.only('status 200: responds with an array of objects with required properties', () => {
+        return request(app)
+        .get('/api/reviews ')
+        .expect(200)
+        .then(({body}) => {
+        const {reviews} = body;
+        expect(reviews).toBeInstanceOf(Array);
```

Comment:
We don't need to explicitly check for an array as we are later checking length and using a forEach which implicitly proves it is an array